frontend/views/containers/dashboard/GroupMembers.vue

```diff
@@ -59,11 +58,6 @@ export default ({
     title: {
       type: String,
       default: L('Members')
-    },
-    action: {
-      type: String,
-      default: 'addMember',
-      validator: (value) => ['addMember'].includes(value)
```

Comment:
as the validator here shows, only available value for this prop is 'addMember' which is also set as the default. So just removed it to clean it up.